ieeeuoft/newhacks-site-2021
# Remove RSVP

## Overview

- Resolves IEEE-49
- Removes RSVP from everywhere it was used:
    - The dashboard
    - Emails that get sent
    - The RSVP view itself
    - Review logic (just sets rsvp=True)

The RSVP field is still left in the model, but there's no real harm since it's totally unused now.

It pains me to remove all of this beautiful code.

## Unit Tests Created

- Removed all test logic for RSVP


## Steps to QA

- Do an application, and look through the dashboard and emails to make sure there's no RSVP stuff in there:
    - Update `REGISTRATION_CLOSED_DATE` in the main settings file to some time in the future so you can actually apply
    - Sign up for an account, submit an application. When you sign up, an email will be printed to the dev console with a link you have to click on to activate your account.
        - Alternatively you can apply with your superuser, but go to http://localhost:8000/admin/auth/user/ first and give yourself a first and last name 
    - Go to the review section of the admin site, and accept yourself by clicking on the team, filling out the fields, and hitting save: http://localhost:8000/admin/review/teamreview/
    - At this point, check the dashboard and make sure it still says application has been submitted, but not decided yet
    - Send decision emails, by clicking the "Send Decisions" button in the top right of the team review page, or going here: http://localhost:8000/admin/review/teamreview/send-mail/. Date range of today is fine unless you did this around midnight, and set status to Accepted
    - Look at the email that gets printed to the console, make sure it looks good
    - Look at the dashboard to make sure it looks good
